README.md

```diff
@@ -1,119 +1,112 @@
-# Project
+# About The Project
 
-> The Data Appliance GX project is intended as a proving ground for GAIA-X and data transfer techologies.
+> The Eclipse DataspaceConnector project is intended as lightweight component to enable sovereign data sharing and policy
+> handling.
 
-# Getting Started
-
-The project requires JDK 11+. To get started:
-
-``` git clone https://github.com/microsoft/Data-Appliance-GX ```
+_Note: items marked with [TBW] indicate that the respective documentation is yet to be written_
 
-``` cd Data-Appliance-GX ```
+## Built with
 
-```./gradlew clean shadowJar```
+One of the guiding principles in developing the connector is simplicity and keeping a small footprint with as little
+external dependencies as possible. We do not want to force any third-party dependencies onto our users, so we aim to
+avoid big frameworks like [Apache Commons](https://commons.apache.org/), [Google Guava](https://github.com/google/guava)
+or the [Spring framework](https://spring.io/). The connector is a plain Java application built with Gradle.
 
-To launch the runtime and client from the root build directory, respectively:
+# Getting Started
 
-```java -jar runtime/build/libs/edc-runtime.jar```
+## Checkout and build code
 
-```java -jar client/build/libs/edc-client.jar```
+The project requires JDK 11+. To get started:
 
-# Build Profiles
+``` shell 
+git clone git@github.com:eclipse-dataspaceconnector/DataSpaceConnector.git
 
-The runtime can be configured with custom modules be enabling various build profiles.
+cd DataSpaceConnector ```
 
-By default, no vault is configured. To build with the file system vault, enable the security profile:
+./gradlew clean build
+```
 
-```./gradlew -Dsecurity.type=fs clean shadowJar ```
+That will build the connector and run unit tests.
 
-The runtime can then be started from the root clone directory using:
+## Run your first connector
 
-``` java -Dedc.vault=secrets/edc-vault.properties -Dedc.keystore=secrets/edc-test-keystore.jks -Dedc.keystore.password=test123 -jar runtime/build/libs/edc-runtime.jar ```
+Connectors can be started using the concept of "launchers", which are essentially compositions of modules defined as
```

Comment:
we need to indroduce the concept of "modules" first. 
what is a "module", what an "extension", a "runtime", a "launcher"...

Reply:
"Modules" refers to "java modules". Also, "launchers" are explained briefly.
But i would expect a more comprehensive documentation from the fully-fledged documentation, that's why there is a `[TBW]` link in the README regarding extensions.